Ballerina-Training/Ballerina-Forum-Application/back-end/bal-forum/main.bal

```diff
@@ -0,0 +1,222 @@
+import ballerina/http;
+import ballerina/sql;
+import ballerina/task;
+import ballerina/time;
+import ballerina/uuid;
+
+configurable int PORT = 4000;
+listener http:Listener forumServerEP = new (PORT);
```

Comment:
Can't this be in-line?

Reply:
+1

Reply:
Fixed via [1eae7ab](https://github.com/ballerina-guides/training-content/pull/12/commits/1eae7abbc8d1d6bdd3a677b4fac348875ee53fb7)